tests/chunkers/test_text.py

```diff
@@ -0,0 +1,46 @@
+# ruff: noqa: E501
```

Comment:
Any specific reason to add this? :D

Reply:
otherwise it complains about all the doc strings.